packages/test/test-end-to-end-tests/src/test/benchmark/opCriticalPath.time.spec.ts

```diff
@@ -19,19 +20,21 @@ import {
 	timeoutPromise,
 } from "@fluidframework/test-utils/internal";
 
-const testContainerConfig: ITestContainerConfig = {
-	runtimeOptions: {
-		enableGroupedBatching: true,
-		summaryOptions: {
-			initialSummarizerDelayMs: 0, // back-compat - Old runtime takes 5 seconds to start summarizer without thi
-			summaryConfigOverrides: {
-				...DefaultSummaryConfiguration,
-				...{ maxOps: 10, initialSummarizerDelayMs: 0, minIdleTime: 10, maxIdleTime: 10 },
-			},
+const runtimeOptions: IContainerRuntimeOptionsInternal = {
```

Comment:
Shouldn't be necessary, `ITestContainerConfig.runtimeOptions` is already the internal type

Reply:
Added changes